godotproject/Player.gd

```diff
@@ -526,14 +526,25 @@ func death_animation_over():
 	# Go back to title screen
 	Global.fade_to_scene(Global.main_menu_path)
 
+func collect(item):
+	var CollectFunctions = ['collect_star_piece', 'collect_health_bug']
+	if (item.COLLECT_ID < CollectFunctions.size()):
+		call(CollectFunctions[item.COLLECT_ID], item)
+
 func collect_star_piece(star_piece):
 	# Increment star piece counter
 	star_piece_count += 1
-	print("Star piece count: ", star_piece_count)
 	# Play sound
 	$CollectStarPieceSoundPlayer.play(0)
 	# Delete star piece
 	star_piece.queue_free()
+	
+func collect_health_bug(health_bug):
+	if (health < 5):
```

Comment:
I would change this to a player const HEALTH_MAX, easier for hard/easy mode implementation